### ISSUE TITLE
Identify audit roles/specializations

### ISSUE THREAD

**Issue by kristenjernigan** [usds/ditap-curriculum-update#69]
**Dependencies**

- We've had a conversation (or have previously shared) around where our specialties fall (audit roles meeting)
- We know who can do what or who wants to do which part of the audit
- Depends on what the rubric looks like

**Definition of Success**

- We've identified SMEs/team members for audit categories
- We've assigned SMEs/team members to align with their specialty

**Other information**

- Needs to be completed before ticket #55

**Comment by kellysmith1008** [comment]
I've created [a spreadsheet of all course-linked content](https://docs.google.com/spreadsheets/d/1e7EW1paqYwzJeoQIWuWF3xYoqdcOy2mVokFnV5QhZWI/edit?usp=sharing) from the [DITAP curriculum](https://ditap.learning-transformation.com/courses/course-v1:ICF+DITAP01+2018_T1/f74f6baf4a0d42558c13bd70b76f03e5/). The rest of today and into tomorrow I'll work on putting this in a voting mechanism for people to select which sections they want to lead in auditing.

**Comment by kellysmith1008** [comment]
We will use a [google form](https://forms.gle/vUoQXGHdHeeNr4Mz8) to collect responses around which roles everyone wants for the audit and the content they'll be responsible for auditing.

**Comment by teresahernandezidt** [comment]
Feedback for the [Google form](https://docs.google.com/document/d/1iOpWll4yIUuWE67PNgCgngtYunj3IOEahOLn-MwL2G8/edit?tab=t.0 is being included from a sync @Chianti and I had on 10/24.

**Comment by teresahernandezidt** [comment]
The following post is feedback we received from Erin on 10/29:

<html>
<body>
<div class="aju" style="cursor: pointer; float: none; padding: 0px 16px; align-items: center; display: flex; height: 80px; min-width: 40px; color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="aCi" style="position: relative;"><img id=":oa_7-e" name=":oa" src="https://lh3.googleusercontent.com/a-/ALV-UjVOB2y6-GmBGmoPlp_PoL5NTHu4cihZ7tBLRpPShTme7Zex0VI=s40-p" class="ajn" jid="erin@expinstitute.com" data-hovercard-id="erin@expinstitute.com" data-name="Erin Murphy" aria-hidden="true" style="width: 40px; height: 40px; border-radius: 50%; display: block; background-color: rgb(164, 194, 244);"></div></div><div class="gs" style="margin: 0px; min-width: 0px; padding: 0px 0px 20px; width: initial; color: rgb(34, 34, 34); font-family: &quot;Google Sans&quot;, Roboto, RobotoDraft, Helvetica, Arial, sans-serif; font-size: 16px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; white-space: normal; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="gE iv gt" style="font-size: 0.875rem; padding: 20px 0px 0px; cursor: pointer;">
Erin Murphy | Erin Murphy | Tue, Oct 29, 2:55 PM (1 day ago) |   |  
-- | -- | -- | -- | --
Erin Murphy
to me, chianti.lomax | to me, chianti.lomax
to me, chianti.lomax

</div><div id=":3pa"><div class="qQVYZb"></div><div class="utdU2e"></div><div class="lQs8Hd" jsaction="SN3rtf:rcuQ6b" jscontroller="i3Ohde"></div><div class="wl4W9b" jsaction="LNSvUb:.CLIENT;xSdBYb:.CLIENT;CDWmBe:.CLIENT;EtHLdc:.CLIENT;pQnh7:.CLIENT;pKHw7e:.CLIENT;Z03mxd:.CLIENT;NZLNxf:.CLIENT;bXglpe:.CLIENT;mzh2Bc:.CLIENT" style="display: flex;"></div></div><div class=""><div class="aHl" style="margin-left: -38px;"></div><div id=":3pm" tabindex="-1"></div><div id=":3pc" class="ii gt adO" jslog="20277; u014N:xr6bB; 1:WyIjdGhyZWFkLWE6ci0yODM0NjU3MjQxMjcyOTAzMTY2Il0.; 4:WyIjbXNnLWY6MTgxNDI3NTY5Nzg4NDU4NDQ3OSIsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLG51bGwsbnVsbCxudWxsLDBd" style="direction: ltr; margin: 8px 0px 0px; padding: 0px; position: relative; font-size: 0.875rem; overflow-x: hidden;"><div id=":3pb" class="a3s aiL " style="direction: ltr; font: small / 1.5 Arial, Helvetica, sans-serif; overflow: auto hidden; position: relative;"><div dir="ltr">Hi Teresa! Hi Chianti! <div><br></div><div>I'm glad you reached out. As I reviewed it, here are some of the things that came to mind. Happy to hop on a call to discuss further! </div><div><br></div><div>I'm excited about the progress on the rubric overall. And I like how you've split it out by role. The two categories you have under Experiential Learning are important for auditing each release, for sure. <br></div><div><br></div><div>I wonder if "how this is applicable' clear to the student?" captures the full scope of the "applicability" bucket. And, maybe it's semantics, but we may want to go one step further from '<i>supporting</i> the key skills and behaviors used in the job' to<span> </span><i>developing</i>. And not just '<i>relating<span> </span></i>to the learner's day-to-day job or future tasks' but<span> </span><i>practicing. </i><br></div><div><br></div><div>You probably already know this, at its most basic level, experiential learning can be thought of in four stages, so I'd be looking to make sure there are opportunities for all of them throughout the course, though it may not make sense to audit each and every release for them as I don't want to lose the forest for the trees. Those stages are:<br></div><div><br></div><div><ul><li style="margin-left: 15px;"><strong>Experience</strong><span> </span>– Engaging in a concrete activity or task.</li><li style="margin-left: 15px;"><strong>Reflect</strong><span> </span>– Observing and reflecting on the experience and your reactions.</li><li style="margin-left: 15px;"><strong>Conceptualize</strong><span> </span>– Drawing insights and forming general principles or conclusions from the reflection.</li><li style="margin-left: 15px;"><strong>Experiment</strong><span> </span>– Applying what you've learned in new situations to test and refine your understanding.</li></ul><div><br></div>Since experiential learning is top-of-mind for aspects of the course such as Live Digital Assessments, Case Studies, and the Capstone, I wonder if they should get some additional treatment? When I think about reviewing case studies outside of DITAP, I look at things like:</div><div><ul><li style="margin-left: 15px;">How realistic is the scenario?</li><li style="margin-left: 15px;">Are the participants asked to step into the role of decision makers (identifying issues, exploring multiple paths and the impact of different choices, prioritizing options, engaging multiple stakeholders, asking additional questions, proposing experiments/solutions)? </li><li style="margin-left: 15px;">Are there opportunities for feedback and incorporating that feedback along the way?</li><li style="margin-left: 15px;">Does it engage the learner emotionally?</li></ul>When I look at any curriculum, I'd be looking at the structure of the activities (solo work, pairs, group discussion) and whether there are different types of reflection prompts and tools that engage people in different ways throughout. I am not sure if it makes sense to be going this deep at this point in the audit or whether we table that to curriculum/instructional design phase. What do you think?<br></div><div><br></div><div>Other questions on my mind:</div><div><ul><li style="margin-left: 15px;">Out of curiosity, what are the higher order skills that the course tries to develop outside of the Learning Objectives? I'm thinking about problem-solving, critical thinking, collaboration? </li><li style="margin-left: 15px;">Farooq asked how do we tie in the LDD? It sounds like we'll need to create a gap analysis report at some point. You all may have already planned for that in one of the upcoming sprints, but I wasn't sure of the answer so I thought I'd raise it here.</li></ul><div>Whew, that was A LOT! Please let me know if this is the kind of feedback you were looking for. I hope I didn't send you off track! If you want to chat more, I have some availability Thursday and Friday this week.</div></div><div><br></div><div>All best,</div><div>Erin</div></div></div></div></div></div>
</body>
</html>
